app/dashboard/static/js/app/view-builds-job-branch-kernel.2017.7.1.js

```diff
@@ -326,6 +329,13 @@ require([
                 archLabelNode.appendChild(document.createTextNode(arch));
             }
 
+            hNode.insertAdjacentHTML(
+                'beforeend', '&nbsp;&dash;&nbsp;');
+            compilerLabelNode = hNode.appendChild(
+                document.createElement('span'));
+            compilerLabelNode.setAttribute('class', 'compiler-label');
```

Comment:
Please use `build-env-label` class which is defined in the CSS.

Reply:
done!